ulib/axstarry/src/syscall_fs/fs_syscall_id.rs

```diff
@@ -15,6 +15,7 @@ numeric_enum_macro::numeric_enum! {
 pub enum FsSyscallId {
     // fs
     GETCWD = 17,
+    EVENT_FD = 19,
```

Comment:
The `EVENT_FD` should be defined in the x86_64 syscall ID and here at the same time, cause x86_64 architecture has `eventfd` and `eventfd2`, otherwise [this line](https://github.com/binary-bruce/arceos/blob/e975ef415888ed49b29a0a016ddb4949d59679b2/ulib/axstarry/src/syscall_fs/mod.rs#L16C4-L16C5)  may issue a warning under the x86_64 (undefined enumeration value for `EVENT_FD`).